package.json

```diff
@@ -12,9 +12,13 @@
 		"lint-js": "eslint js/",
 		"precommit": "npm run lint-css && npm run lint-js",
 		"prod": "cross-env NODE_ENV=production webpack --mode production --progress",
-		"update-deps": "rm -rf node_modules && rm package-lock.json && npm install"
+		"update-deps": "rm -rf node_modules && rm package-lock.json && npm install",
+		"rename": "./bin/rename.js"
```

Comment:
I think in place of `rename` `init` would more sense here.

Reply:
`init` is fine, but in this script we only renaming the strings. In the `init` method we can add `npm run rename && npm install` command which actually do the `init` work.